api/graphql/schema.py

```diff
@@ -37,9 +43,17 @@ class AllowAuthenticatedFilter(AuthFilter):
     permission_classes = (AllowAuthenticated,)
 
 
-class ReservationUnitsFilter(AuthFilter):
+class ReservationUnitsFilter(AuthFilter, django_filters.FilterSet):
     permission_classes = (ReservationUnitPermission,)
 
+    unity = ModelInFilter(field_name="unit", queryset=Unit.objects.all())
```

Comment:
Unity?

Reply:
:see_no_evil:  typo => i'll fix

Reply:
oh, actually this is completely unnecessary here => was testing FilterSets with bad results => I'll remove this `FilterSet` experiment here

Reply:
Okay, actually got the `FilterSet` class to work.